README.md

```diff
@@ -5,6 +5,16 @@ This is the website for the DANDI project.
 This website is powered by Jekyll and some Bootstrap, Bootwatch. Go to 
 *aboutwebsite.md*  to learn how to copy and modify this page for your purpose. 
 
+## Local development
+
+Test your changes locally by running the following command:
+
+```bash
+docker run --rm -it --volume="$PWD:/srv/jekyll" \
+  --volume="$PWD/vendor/bundle:/usr/local/bundle"  \
+  --env JEKYLL_ENV=development -p 4000:4000 \
+  jekyll/jekyll:4.0.1  jekyll serve --config _config.yml
+```
```

Comment:
running from top directory on Linux I got

```shell
❯ docker run --rm -it --volume="$PWD:/srv/jekyll" \
  --volume="$PWD/vendor/bundle:/usr/local/bundle"  \
  --env JEKYLL_ENV=development -p 4000:4000 \
  jekyll/jekyll:4.0.1  jekyll serve --config _config.yml
Unable to find image 'jekyll/jekyll:4.0.1' locally
4.0.1: Pulling from jekyll/jekyll
cbdbe7a5bc2a: Pull complete 
aa8ae8202b42: Pull complete 
b21786fe7c0d: Pull complete 
2e8deee27474: Pull complete 
b84a6ba69f44: Pull complete 
593a81637df6: Pull complete 
Digest: sha256:2500fafd64afad32e40295c23a7054d95be95b2cf5b76b285af2a2149e2d9322
Status: Downloaded newer image for jekyll/jekyll:4.0.1
Could not locate Gemfile

❯ find -iname Gemfile
./Gemfile
find: ‘./.sass-cache’: Permission denied
find: ‘./_site’: Permission denied
```

Reply:
the docker command needs a working directory flag to be in `/srv/jekyll`